script/cadence/contracts/FlowColdStorageProxy.cdc

```diff
@@ -44,9 +59,11 @@ pub contract FlowColdStorageProxy {
         pub var lastNonce: Int64
         access(self) let flowVault: @FungibleToken.Vault
         access(self) let publicKey: [UInt8]
+		access(self) let nodeDelegator: @FlowIDTableStaking.NodeDelegator
 
-        init(publicKey: [UInt8]) {
+        init(publicKey: [UInt8], stakingNodeID: String) {
```

Comment:
It is right to register a delegator for the user by default, or should there be a different method to allow them the option to register only if they want to?